lib/conditionals.md

```diff
@@ -5,8 +5,8 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional is a comparison made between two arguments to determine whether it is true or false. These usually begin as if or else statements. 
```

Comment:
> A conditional is a comparison made between two arguments

Is a conditional always a _comparison_? If not, what is a more universal definition of what a conditional is?

You also wrote that conditionals

> ...begin as if or else statements.

What do you mean by "begin as"? Do you mean that a conditional starts off as an `if` statement and later becomes something else? Please clarify.

Additionally, what are other cases for using a conditional statement?